docs/Azure_IoT_Hub_integration/Device_operations/Upgrading_firmware.md

```diff
@@ -0,0 +1,117 @@
+# Upgrading device firmware
```

Comment:
Do you consider writing this in a more tutorial way? For example - Let's do a firmware upgrade:
1. Schedule firmware 
2. Check the result 
3. Check Coiote that task is scheduled
4. Check on the device that firmware is in the newest version etc.

Reply:
Changed the doc structure and added a section - please check if this is what you had in mind